### PR TITLE
added a new command

### PR DESCRIPTION
**Pull request by Dhruvin12**

### CODE REVIEW

**Comment by Dekriel** [comment]
my first thought is that it looks a bit messy.

**Comment by Dhruvin12** [comment]
🤡 maybe you can do some changes

**Comment by Dekriel** [comment]
ok

**Review by sifte — CHANGES_REQUESTED** [review]
emoji's wont work

**Review by sifte — APPROVED** [review]

File: src/cogs/mod.py [hunk 1]
```diff
@@ -270,5 +270,30 @@ async def delwarn(
 
         await ctx.send('Warn was successfully deleted.')
 
+    @commands.command()
+    @commands.guild_only()
+    async def whois(self, ctx, member: discord.Member = None):
+        member = member or ctx.author
+        embed = discord.Embed(title=f"Whois {member.name}",timestamp=ctx.message.created_at,description=f"Here is some information about {member.mention}.")
+        embed.set_thumbnail(url=member.avatar_url)
```
Comment (sarthak-py) [inline src/cogs/mod.py:278]: outdated `avatar_url`

**Review by sarthak-py — CHANGES_REQUESTED** [review]

File: src/cogs/mod.py (hunk 1 above)
Comment (sarthak-py) [reply src/cogs/mod.py:278]: you might like using `display_avatar` mb

**Review by sarthak-py — APPROVED** [review]

**Pull request closed without merging**